styleguide/assets/variables.css

```diff
@@ -37,6 +37,8 @@
 	--color-box-shadow: rgba(77, 77, 77, 0.5);
 	--color-border: #ededed;
 	--color-border-dark: #dbdbdb;
+	--color-main-background-semitransparent: rgba(255,255,255,0.8);
```

Comment:
We already have a variable for this actually, and it’s used in Dashboard: `var(--color-background-translucent)`

Reply:
This will also ensure it works with dark mode.

Reply:
this is just for documentation, we get the variables from theming now